tsl/test/expected/compression_update_delete-16.out

```diff
@@ -2844,15 +2844,15 @@ EXPLAIN (costs off) DELETE FROM test_partials WHERE time >= ALL(SELECT time from
 DELETE FROM test_partials WHERE time >= ALL(SELECT time from test_partials);
 -- All 3 chunks will now become partially compressed chunks
 EXPLAIN (costs off) SELECT * FROM test_partials ORDER BY time;
-                                     QUERY PLAN                                      
--------------------------------------------------------------------------------------
+                                                     QUERY PLAN                                                     
+--------------------------------------------------------------------------------------------------------------------
  Custom Scan (ChunkAppend) on test_partials
    Order: test_partials."time"
    ->  Merge Append
          Sort Key: _hyper_35_68_chunk."time"
          ->  Custom Scan (DecompressChunk) on _hyper_35_68_chunk
                ->  Sort
-                     Sort Key: compress_hyper_36_71_chunk._ts_meta_sequence_num DESC
+                     Sort Key: compress_hyper_36_71_chunk._ts_meta_min_1, compress_hyper_36_71_chunk._ts_meta_max_1
```

Comment:
should there be an explicit DESC also in the new sort key? Or should it be ASC?

Reply:
Previous sort was a DESC becuase compression is set to "time DESC" so to get ASC ordering, you needed reverse order on the sequence number. New ordering should always mirror the column ordering because we are using actual values instead of a sequence which can have its own ordering.